### DISCUSSION TITLE
Mlflow inference for image

### DISCUSSION THREAD

**Discussion by Hir98** [mlflow/mlflow#9422] [Q&A]
Hello

I want to know that can we use mlflow inference for images??
To serve yolov8(object detection) model through mlflow tracking is it possible ??
If possible then please attached some reference for that.
Can you please help me .

**Comment by sniafas** [comment]
Absolutely, mlflow can be use to inference images.
You can design your MLflow model (Pyfunc) to map image urls or image files from the POST request and perform the inference in pyfunc model while you have already loaded yolo.
